src/sonic-yang-models/yang-models/sonic-fabric-monitor.yang

```diff
@@ -32,17 +32,31 @@ module sonic-fabric-monitor{
                 }
 
                 leaf monPollThreshIsolation {
-                    type uint32;
+                    type uint8;
```

Comment:
why do you change type here?

Reply:
same question also on `leaf monPollThreshRecovery {`

Reply:
> why do you change type here?
I am expecting this number be very small, like 1,2, or something, so I changed it to uint8 . Let me know if you prefer leave it as orginal uint32.

Reply:
If the number is really small, can you add range?

Reply:
> If the number is really small, can you add range?

added